<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>163</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Results folder</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="resultsFolder"/>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="resultsFolderBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>gcc command</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="gcc"/>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="gccBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>clang command</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="clang"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="clangBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>cppcheck command</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="cppcheck"/>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="cppcheckBrowse">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>2</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>10</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>460</x>
     <y>143</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clangBrowse</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>clangBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cppcheckBrowse</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>cppcheckBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>gccBrowse</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>gccBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>resultsFolderBrowse</sender>
   <signal>clicked()</signal>
   <receiver>SettingsDialog</receiver>
   <slot>resultsFolderBrowse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>16</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>resultsFolderBrowse()</slot>
  <slot>clangBrowse()</slot>
  <slot>cppcheckBrowse()</slot>
  <slot>gccBrowse()</slot>
 </slots>
</ui>
